tests/auto/ci_auto.py

```diff
@@ -195,11 +200,11 @@ def run(self):
     def send_comment_text(self):
         logger = logging.getLogger('JOB/SEND_COMMENT_TEXT')
         logger.info(f'Comment Text: {self.comment_text}')
-        self.comment_text_append('Please make changes and add '
-                                 'the following label back:')
-        self.comment_text_append(f'{self.machine}'
-                                 f'-{self.compiler}'
-                                 f'-{self.preq_dict["action"]}')
+        self.comment_append('If test failed, please make changes and add '
+                            'the following label back:')
+        self.comment_append(f'ci-{self.machine}'
+                            f'-{self.compiler}'
+                            f'-{self.preq_dict["action"]}')
```

Comment:
Does this system remove the label from the PR right now?

Reply:
The code is set up to remove the label, and I have used that capability (so I know it works), but for convenience I usually comment out the line that deletes the label when I move the code to an HPC. That way I don't have to add it back, and for repos where I don't have permissions, I don't have to ask someone else to add it back.